crates/rpc/rpc/src/admin.rs

```diff
@@ -12,12 +12,14 @@ use reth_rpc_types::{NodeInfo, PeerEthProtocolInfo, PeerInfo, PeerNetworkInfo, P
 pub struct AdminApi<N> {
     /// An interface to interact with the network
     network: N,
+    /// The specification of the blockchain's configuration.
+    chain_spec: ChainSpec,
```

Comment:
this should be Arced
```suggestion
    chain_spec: Arc<ChainSpec>,
```

Reply:
Done